tests/src/Unit/AvailabilityManagerTest.php

```diff
@@ -90,7 +97,9 @@ public function testCheck() {
     $result = $this->availabilityManager->check($entity, 1);
     $this->assertFalse($result, 'The checked entity is not available when a checker that returns FALSE applies');
 
-    $expectedCheckers = [$first_checker, $second_checker, $third_checker, $fourth_checker];
```

Comment:
This also fails the validator because the line is longer than 80. @haringsrob and @bojanz What do you want to do about it?